definitions/marts/looker_studio_marts/npq/ls_npqeyl_enrolments.sqlx

```diff
@@ -9,7 +9,85 @@ config {
     },
     description: "This mart provides a report of all Early Years Leadership NPQs (NPQEYL). The table links to fields from GIAS and Ofsted establishments and feeds into the NPQEYL dashboard.",
     columns: {

```

Comment:
Is this NPQ enrolments plus some bonus fields?

Reply:
Would recommend updating the fields below for any field you update in NPQ enrolments